spec/requests/api/v1/course_spec.rb

```diff
@@ -46,74 +46,103 @@
       )
     end
 
-    it "returns http success" do
-      get "/api/v1/courses", headers: { 'HTTP_AUTHORIZATION' => ActionController::HttpAuthentication::Token.encode_credentials("bats") }
-      expect(response).to have_http_status(:success)
-    end
+    expected_json = [
+      {
+        "course_code" => "2HPF",
+        "start_month" => "2019-09-01T00:00:00Z",
+        "start_month_string" => "September",
+        "name" => "Religious Education",
+        "study_mode" => "F",
+        "copy_form_required" => "Y",
+        "profpost_flag" => "PG",
+        "program_type" => "SD",
+        "age_range" => "P",
+        "modular" => "",
+        "english" => 3,
+        "maths" => 9,
+        "science" => nil,
+        "qualification" => 1,
+        "campus_statuses" => [
+          {
+            "campus_code" => "-",
+            "name" => "Main Site",
+            "vac_status" => "F",
+            "publish" => "Y",
+            "status" => "R",
+            "course_open_date" => "2018-10-09"
+          }
+        ],
+        "subjects" => [
+          {
+            "subject_code" => "1",
+            "subject_name" => "Secondary"
+          },
+          {
+            "subject_code" => "2",
+            "subject_name" => "Mathematics"
+          }
+        ],
+        "provider" => {
+          "institution_code" => "2LD",
+          "institution_name" => "ACME SCITT",
+          "institution_type" => "B",
+          "accrediting_provider" => 'N',
+          "address1" => "Sydney Russell School",
+          "address2" => "Parsloes Avenue",
+          "address3" => "Dagenham",
+          "address4" => "Essex",
+          "postcode" => "RM9 5QT",
+          "region_code" => "07",
+          "scheme_member" => "Y",
+        },
+        "accrediting_provider" => nil
+      }
+    ]
+
+    describe 'index' do
+      it "returns http success" do
+        get "/api/v1/courses", headers: { 'HTTP_AUTHORIZATION' => ActionController::HttpAuthentication::Token.encode_credentials("bats") }
+        expect(response).to have_http_status(:success)
+      end
+
+      it "returns http unauthorised" do
+        get "/api/v1/courses", headers: { 'HTTP_AUTHORIZATION' => ActionController::HttpAuthentication::Token.encode_credentials("foo") }
+        expect(response).to have_http_status(:unauthorized)
+      end
 
-    it "returns http unauthorised" do
-      get "/api/v1/courses", headers: { 'HTTP_AUTHORIZATION' => ActionController::HttpAuthentication::Token.encode_credentials("foo") }
-      expect(response).to have_http_status(:unauthorized)
+      it "JSON body response contains expected course attributes" do
+        get "/api/v1/courses", headers: { 'HTTP_AUTHORIZATION' => ActionController::HttpAuthentication::Token.encode_credentials("bats") }
+        json = JSON.parse(response.body)
+        expect(json). to eq expected_json
+      end
+
+      it "returns http success" do
```

Comment:
there should be two describe blocks one for index as the default and index#parameters as the other